src/lib.rs

```diff
@@ -74,6 +74,7 @@ use self::unicode::Unicode;
 
 mod ascii;
 mod unicode;
+pub mod serde;
```

Comment:
```suggestion
mod serde;
```

I'm not seeing why this needs to be public. Trait implementations aren't part of a module.